src/routes/GroupDetail/GroupDetail.js

```diff
@@ -213,6 +213,11 @@ class UnwrappedGroupDetail extends Component {
             </div>
           ) : (
             <div styleName='g.groupDescription'>
+              <h3>{t('Purpose')}</h3>
+              <ClickCatcher>
+                <HyloHTML element='span' html={TextHelpers.markdown(group.purpose)} />
```

Comment:
Ahh so we can display fancy text...

Reply:
Yeah I think we decided not to promote fancy text but allow it via markdown